--- origsrc/gtk-sharp-2.12.9/atk/glue/Makefile.am	2009-12-04 00:27:33.091203000 -0600
+++ src/gtk-sharp-2.12.9/atk/glue/Makefile.am	2009-12-04 00:26:19.000000000 -0600
@@ -1,6 +1,6 @@
 lib_LTLIBRARIES = libatksharpglue-2.la
 
-libatksharpglue_2_la_LDFLAGS = -module -avoid-version -no-undefined
+libatksharpglue_2_la_LDFLAGS = -avoid-version -no-undefined
 
 libatksharpglue_2_la_SOURCES = 	\
 	hyperlink.c \
--- origsrc/gtk-sharp-2.12.9/gdk/glue/Makefile.am	2009-12-04 00:27:33.091203000 -0600
+++ src/gtk-sharp-2.12.9/gdk/glue/Makefile.am	2009-12-04 00:26:19.000000000 -0600
@@ -12,7 +12,7 @@ nodist_libgdksharpglue_2_la_SOURCES = ge
 
 # Adding a new glue file?
 
-libgdksharpglue_2_la_LDFLAGS = -module -avoid-version -no-undefined
+libgdksharpglue_2_la_LDFLAGS = -avoid-version -no-undefined
 
 libgdksharpglue_2_la_LIBADD = $(GTK_LIBS)
 
--- origsrc/gtk-sharp-2.12.9/glade/glue/Makefile.am	2009-12-04 00:27:33.106803100 -0600
+++ src/gtk-sharp-2.12.9/glade/glue/Makefile.am	2009-12-04 00:26:19.000000000 -0600
@@ -6,7 +6,7 @@ else
 TARGET = 
 endif
 
-libgladesharpglue_2_la_LDFLAGS = -module -avoid-version -no-undefined
+libgladesharpglue_2_la_LDFLAGS = -avoid-version -no-undefined
 
 libgladesharpglue_2_la_SOURCES = 	\
 	gladexml.c
--- origsrc/gtk-sharp-2.12.9/glib/glue/Makefile.am	2009-12-04 00:27:33.106803100 -0600
+++ src/gtk-sharp-2.12.9/glib/glue/Makefile.am	2009-12-04 00:26:19.000000000 -0600
@@ -1,6 +1,6 @@
 lib_LTLIBRARIES = libglibsharpglue-2.la
 
-libglibsharpglue_2_la_LDFLAGS = -module -avoid-version -no-undefined
+libglibsharpglue_2_la_LDFLAGS = -avoid-version -no-undefined
 
 libglibsharpglue_2_la_SOURCES = 	\
 	closure.c		\
--- origsrc/gtk-sharp-2.12.9/gtk/glue/Makefile.am	2009-12-04 00:27:33.106803100 -0600
+++ src/gtk-sharp-2.12.9/gtk/glue/Makefile.am	2009-12-04 00:26:19.000000000 -0600
@@ -19,7 +19,7 @@ nodist_libgtksharpglue_2_la_SOURCES = ge
 
 # Adding a new glue file?
 
-libgtksharpglue_2_la_LDFLAGS = -module -avoid-version -no-undefined
+libgtksharpglue_2_la_LDFLAGS = -avoid-version -no-undefined
 
 libgtksharpglue_2_la_LIBADD = $(GTK_LIBS)
 
--- origsrc/gtk-sharp-2.12.9/pango/glue/Makefile.am	2009-12-04 00:27:33.122403100 -0600
+++ src/gtk-sharp-2.12.9/pango/glue/Makefile.am	2009-12-04 00:26:19.000000000 -0600
@@ -8,7 +8,7 @@ nodist_libpangosharpglue_2_la_SOURCES = 
 
 # Adding a new glue file?
 
-libpangosharpglue_2_la_LDFLAGS = -module -avoid-version -no-undefined
+libpangosharpglue_2_la_LDFLAGS = -avoid-version -no-undefined
 
 libpangosharpglue_2_la_LIBADD = $(PANGO_LIBS)
 
